samples/core/xgboost_training_cm/xgboost_training_cm.py

```diff
@@ -203,8 +203,8 @@ def dataproc_predict_op(
     description='A trainer that does end-to-end distributed training for XGBoost models.'
 )
 def xgb_train_pipeline(
-    output='gs://your-gcs-bucket',
-    project='your-gcp-project',
+    output,
```

Comment:
Do you think that this might make the `output` parameter more confusing? It would be less confusing if it was `output_gcs_uri`.

Typing the input is another good option. Unfortunately, the Frontend does not surface the type information,even though the SDK makes it available.

Reply:
+1 to surfacing the description to the UI.
I'm not a very big fan to leaving it blank, either... For example, if a user did not go through the doc of this sample, (s)he might be confused about what to put for the output. Should it be with the prefix `gs://` or without? Does it support other storage platform other than GCS?

IMO a placeholder is a very self-explanatory way to tell user what should be put here.

Reply:
OK I just think non blank field is less discoverable. I don't have strong opinion. 
Surface type info in UI would be very nice.